### ISSUE TITLE
Do not clear output area when switching tabs/panes [feature request]

### ISSUE THREAD

**Issue by ondrejtichacek** [nteract/hydrogen#1539]
**Current behavior:**
After running a file A, its outputs are shown in the hydrogen output area. Currently, if you switch a tab/pane to a file B, the output area shows data from a kernel that belongs to the file B, or it shows "no output to display".

**Proposal:**
For my workflow, it would be desirable to be able to see the **last active output area** (belonging to a file A) if there is no running kernel belonging to file B (and the output area would read "no output to display"). There could be a small warning sign in such a case - noting that this output area belongs in fact to a different file.

**Example situation:**
Debugging program: I run script in file A - error points to file B. I switch to file B, find the line, but I no longer see the details of the error, unless I switch to file A again.

**Note:** I am aware that something similar can be achieved when using global kernel. However, that is something, I would like to avoid, since the ability to run multiple kernels at the same time is very useful.

**Comment by ondrejtichacek** [comment]
Seeing the reference in #1542, I think I should clarify: the current behavior does not seem to be a bug - switching files A - B - A does not permanently clear the output area of A. This feature request is aimed at showing output of A in the output area of B in a case that B has no active kernel.

**Comment by kylebarron** [comment]
@ondrejtichacek 	you're referring to the output pane, not the bubbles inline?

**Comment by ondrejtichacek** [comment]
@kylebarron That is correct, I am talking about the output pane, by default placed on the right side of atom interface. I made a [short video](https://arctg.cz/a/github-hydrogen-issue1542.webm) capturing the issue with my workflow.

**Comment by wadethestealth** [comment]
I don't believe this should be default behavior, but I could see where this could be useful.
